testing/src/test/TestManifest.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  package="org.oppia.testing">
+  <uses-sdk android:minSdkVersion="16"
```

Comment:
min SDK should be 19, and we should target 29 not specify a max.

Reply:
Done! Quick note though: I just noticed that some of the tests in utility were not passing with this error: 
```
java.lang.IllegalArgumentException: API level 29 is not available
```
So, I changed utility's test manifest to target 28 and this fixed the issue.